openedx/core/djangoapps/programs/rest_api/v1/views.py

```diff
@@ -0,0 +1,361 @@
+"""Views for the Programs REST API v1."""
+
+import logging
+
+from enterprise.models import EnterpriseCourseEnrollment
```

Comment:
I know we're mostly just moving code around to help support a pending DEPR. It would be nice if we're touching some of these files to remove the direct imports of models from other Django apps.

If we don't fix this here and now, can we at least get a ticket in the backlog to remove the direct import and use of the `EnterpriseCourseEnrollment` and `CourseEnrollment` models in this file?

Reply:
I'm working on cleanup tickets now, and that would be a great cleanup ticket, so I will go ahead and create that right away. Nice catch.